src/Http/Controllers/Admin/TranslationsController.php

```diff
@@ -67,4 +84,77 @@ public function update(UpdateTranslation $request, Translation $translation)
         return redirect('admin/translation');
     }
 
+    /**
+     * @param UpdateTranslation $request
+     * @return \Symfony\Component\HttpFoundation\BinaryFileResponse
+     */
+    public function export(UpdateTranslation $request)
+    {
+        $currentTime = Carbon::now()->toDateTimeString();
+        $nameOfExportedFile = 'translations' . $currentTime . '.xlsx';
+        return Excel::download(new TranslationsExport($request), $nameOfExportedFile);
+    }
+
+    /**
+     * @param ImportTranslation $request
+     * @return array|\Illuminate\Http\JsonResponse|mixed
+     */
+    public function import(ImportTranslation $request)
+    {
+        if ($request->hasFile('fileImport')) {
+            $chooseLanguage = strtolower($request->importLanguage);
+
+            if ($request->file('fileImport')->getClientOriginalExtension() != "xlsx")
+                return response()->json("Unsupported file type", 409);
+
+            try {
+                $collection = (new TranslationsImport())->toCollection($request->file('fileImport'))->first();
+            } catch (\Exception $e) {
+                return response()->json("Unsupported file type", 409);
+            }
+
+            $requiredHeaders = ['namespace', 'group', 'default', $chooseLanguage];
+
+            foreach ($requiredHeaders as $item) {
+                if (!isset($collection->first()[$item])) return response()->json("Wrong syntax in your import", 409);
+            }
```

Comment:
Till here, and if some error occurs, throw exception, here you can then, just call that method and add try catch.